commands.py

```
@@ -20,9 +23,121 @@ def __init__(self, bot):
         Args:
             bot: The discord bot
         """
+
         self.bot = bot
         self.datasource = JsonDatasourceHandler()
 
+    @tasks.loop(minutes=5)
+    async def check_streamers(self):
+        """
+        Checks twitch streamers every interval to be able to announce go lives
+
+        Returns:
+            None
+        """
+
+        streamer_mapper = StreamerMapper(self.datasource)
+        streamers = streamer_mapper.map()
+        twitch_handler = TwitchHandler()
```

I don't need to create a new instance of TwitchHandler every 5 minutes, this can be moved to a class property.

Something to consider in the future, what if the TwitchAPI threshold was hit or the API became unavailable temporarily? I should throw an exception if we don't get the desired result and catch it here to prevent the bot/loop from breaking. (I will tackle this in a future branch / make an issue for it)